apps/src/lab2/views/dialogs/ShareDialog.tsx

```diff
@@ -28,65 +28,108 @@ const CopyToClipboardButton: React.FunctionComponent<{
   }, [shareUrl, projectType]);
 
   return (
-    <button
-      type="button"
-      onClick={handleCopyToClipboard}
-      className={moduleStyles.copyToClipboard}
-    >
-      <FontAwesomeV6Icon
-        iconName={copiedToClipboard ? 'clipboard-check' : 'clipboard'}
-        iconStyle="thin"
-        className={moduleStyles.copyToClipboardIcon}
+    <div>
+      <Button
+        iconLeft={{
+          iconName: copiedToClipboard ? 'clipboard-check' : 'clipboard',
+        }}
+        ariaLabel={i18n.copyLinkToProject()}
+        text={i18n.copyLinkToProject()}
+        type="primary"
+        color="black"
+        size="m"
+        onClick={handleCopyToClipboard}
       />
-      {i18n.copyLinkToProject()}
-    </button>
+    </div>
   );
 };
 
 /**
  * A new implementation of the project share dialog for Lab2 labs.  Currently only used
  * by Music Lab and Python Lab, and only supports a minimal subset of functionality.
  */
+
 const ShareDialog: React.FunctionComponent<{
+  id?: string;
```

Comment:
could this be something more specific than `id` to avoid confusion with HTML id? Maybe `dialogId` or `dialogType`? Later on if we have a list of dialog types (like 'hoc2024') in shared constants, we can use that as a type here too

Reply:
Good call.  Updated.